doc/plugins/ip-restriction-cn.md

```diff
@@ -0,0 +1,84 @@
+[English](ip-restriction.md)
+
+# 目录
+- [**名字**](#名字)
+- [**属性**](#属性)
+- [**如何启用**](#如何启用)
+- [**测试插件**](#测试插件)
+- [**禁用插件**](#禁用插件)
+
+## 名字
+
+`ip-restriction` 可以通过以下方式限制对服务或路线的访问，将 IP 地址列入白名单或黑名单。 单个 IP 地址，多个 IP地址 或 CIDR 范围，可以使用类似 10.10.10.0/24 的 CIDR 表示法(将很快支持 IPv6)。
+
+## 属性
+
+* `whitelist`: 可选，加入白名单的IP地址 或 CIDR 范围
+* `blacklist`: 可选，加入黑名单的IP地址 或 CIDR 范围
+
+只能单独启用白名单或黑名单，两个不能一起使用。
+
+## 如何启用
+
+下面是一个示例，在指定的 route 上开启了 ip-restriction 插件:
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+{
+    "uri": "/hello",
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "127.0.0.1:1980": 1
+        }
+    },
+    "plugins": {
+        "ip-restriction": {
+            "whitelist": [
+                "127.0.0.1",
+                "113.74.26.106/24"
+            ]
+        }
+    }
+}'
+```
+
+## 测试插件
+
+访问 `127.0.0.1`:
+
+```shell
+$ curl http://127.0.0.1:9080/index.html
+HTTP/1.1 200 OK
+...
+```
+
+访问 `127.0.0.2`:
+
+```shell
+$ curl http://127.0.0.2:9080/index.html -i
+HTTP/1.1 403 Unauthorized
+...
+{"message":"Your IP address is not allowed"}
```

Comment:
consider adding a space like `{"message": "Your XXX"}` ?

Reply:
it encoded by `cjson` library by machine.